### ISSUE TITLE
Video text track: Kind attribute missing from block markup when value set to Subtitles

### ISSUE THREAD

**Issue by chthonic-ds** [WordPress/gutenberg#26673]
**Describe the bug**
Gutenberg 9.2 introduced text tracks for the Video block. The text tracks UI allows the user to add a value for the Kind attribute. When the value for Kind is set to Subtitles this attribute is not displayed in the block markup output to the browser.

**To reproduce**
1. Add a Video block with a text track: [subtitles.zip](https://github.com/WordPress/gutenberg/files/5483996/subtitles.zip).
2. Set Subtitles as the value for Kind.
3. Save the post and observe the Kind attribute is missing from the `<track>` element.
4. Set Captions as the value for Kind.
5. Observe the Kind attribute is present in the `<track>` element.

All other available values (captions, chapters, descriptions, metadata) are set for the `<track>` element attribute as expected.

**Expected behavior**
The Kind attribute should appear as an attribute of the `<track>` element when Subtitles is set as the value.

**Screenshots**
![text-track-subtitles](https://user-images.githubusercontent.com/29587253/98039306-5fb24600-1e83-11eb-8d0e-210699550fd4.jpg)
![text-track-captions](https://user-images.githubusercontent.com/29587253/98039311-6345cd00-1e83-11eb-8014-a02ddfced7fd.jpg)

**Editor version (please complete the following information):**
- WordPress version: [e.g: 5.5.3]
- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default? Gutenberg plugin
- If the Gutenberg plugin is installed, which version is it? 9.2.2

**Comment by gwwar** [comment]
Testing on WP 5.7 / Gutenberg trunk, I verified that "subtitles" is not set to kind, but that all other values are.

| Subtitles | Captions |
|-----|-----|
| <img width="803" alt="subtitles" src="https://user-images.githubusercontent.com/1270189/111684393-e6890f00-87e3-11eb-9890-b1b97ac58737.png"> | <img width="794" alt="captions" src="https://user-images.githubusercontent.com/1270189/111684464-facd0c00-87e3-11eb-94e0-167752994684.png"> |

**Comment by paaljoachim** [comment]
Testing with WordPress 5.7
Gutenberg 10.1.1
Twenty Twenty One.

Added the video block.
Uploaded a video.
Downloaded the subtitles.zip link mentioned. Unzipped it.
Uploaded the subtitles.vtt file to the video.

<img width="588" alt="Screen Shot 2021-03-18 at 20 15 37" src="https://user-images.githubusercontent.com/5323259/111684231-badb4880-8826-11eb-825f-7efab4867a30.png">

Frontend HTML code:
![Screen Shot 2021-03-18 at 20 17 41](https://user-images.githubusercontent.com/5323259/111684455-055cc500-8827-11eb-8789-3e9e632bd235.png)

-----

I tested switching from captions over to Subtitles.
![Screen Shot 2021-03-18 at 20 18 43](https://user-images.githubusercontent.com/5323259/111684615-3210dc80-8827-11eb-909b-9d5a5ae88a79.png)

Frontend:
![Screen Shot 2021-03-18 at 20 19 01](https://user-images.githubusercontent.com/5323259/111684633-3b01ae00-8827-11eb-9cea-70e37575a3ee.png)

Subtitles are not mentioned in the HTML code.
